spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/Resilience4jBulkheadProvider.java

```diff
@@ -143,11 +143,13 @@ private <T> Supplier<CompletionStage<T>> decorateBulkhead(final String id, final
 
 	private Resilience4jBulkheadConfigurationBuilder.BulkheadConfiguration getConfiguration(String id) {
 		Resilience4jBulkheadConfigurationBuilder builder = new Resilience4jBulkheadConfigurationBuilder();
+		Resilience4jBulkheadConfigurationBuilder.BulkheadConfiguration defaultConfiguration = this.defaultConfiguration
```

Comment:
This is just a minor suggestion really.
The method `configureDefault(..)` allows to set the variable `defaultConfiguration` to `null`.